aws/resource_aws_db_instance_test.go

```diff
@@ -2981,13 +2981,6 @@ func TestAccAWSDBInstance_CACertificateIdentifier(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSDBInstanceDestroy,
 		Steps: []resource.TestStep{
-			{
-				Config: fmt.Sprintf(testAccAWSDBInstanceConfigWithCACertificateIdentifier, "rds-ca-2015"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSDBInstanceExists(resourceName, &dbInstance),
-					resource.TestCheckResourceAttr(resourceName, "ca_cert_identifier", "rds-ca-2015"),
-				),
-			},
 			// Ensure we are able to modify the CACertIdentifier
```

Comment:
Let's also remove this comment, since its no longer accurate. 👍

Reply:
Thanks, done.